wpilib/wpilib/speedcontrollergroup.py

```diff
@@ -7,15 +7,33 @@
 # the project.
 # ----------------------------------------------------------------------------
 from .interfaces.speedcontroller import SpeedController
+from .sendablebase import SendableBase
 
 __all__ = ["SpeedControllerGroup"]
 
 
-class SpeedControllerGroup(SpeedController):
+class SpeedControllerGroup(SendableBase, SpeedController):
+    """Allows multiple :class:`.SpeedController` objects to be linked together"""
+
+    instances = 0
+
     def __init__(self, *args):
+        """Create a new SpeedControllerGroup with the provided SpeedControllers.
+
+        :param args: SpeedControllers to add
+        :type args: :class:`.SpeedController`
+        """
+        SendableBase.__init__(self)
+        SpeedController.__init__(self)
+
+        for speedController in args:
```

Comment:
Upstream requires at least one SpeedController to be passed. Shouldn't we check this? Could do it a similar way upstream does (i.e. specify one explicitly as an argument) - would be the simplest and most fool-proof way.